openreview/conference/templates/programchairWebfield.js

```diff
@@ -455,14 +455,16 @@ var findProfile = function(profiles, id) {
     return profiles[id];
   }
   var profile = _.find(profiles, function(p) {
-    return _.includes(p.allNames, id) || _.includes(p.allEmails, id);
+    return _.includes(_.map(p.allNames, 'username'), id) || _.includes(p.allEmails, id);
```

Comment:
Rather than doing a map, then includes I think it would be better to just do a `find`
```suggestion
    return _.find(p.allNames, ['username', id]) || _.includes(p.allEmails, id);
```

Reply:
Why don’t we save the username only in allNames, is there a reason to have the whole name object?

Reply:
You're right, that would be the better solution. I looked and that field is only ever used for the usernames.